### ISSUE TITLE
indentation with multi-line statements

### ISSUE THREAD

**Issue by haraldkubota** [eslint/eslint#10050]
**Tell us about your environment**

* **ESLint Version:** v4.18.2
* **Node Version:** v8.9.4
* **npm Version:** v5.6.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "env": {
    "browser": false,
    "node": true,
    "commonjs": true,
    "es6": true
  },
  "extends": "eslint:recommended",
  "parserOptions": {
    "ecmaVersion": 8,
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true
    },
    "sourceType": "module"
  },
  "plugins": [
    ],
  "rules": {
    "space-infix-ops": ["error", {"int32Hint": false}],
    "no-unused-vars": ["warn", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }],
    "no-console": "off",
    "indent": [
      "warn",
      2
    ],
    "linebreak-style": [
      "error",
      "unix"
    ],
    "quotes": [
      "error",
      "single"
    ],
    "semi": [
      "warn",
      "never"
    ]
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
function foo(a, b) {}

foo('New box ' + 'boxName' +
  ' created. What would you like to add to the box?',
  'Anything to put in?')
```

```bash
eslint src/test.js 
```

**What did you expect to happen?**

No complains about line 5 (the last line) as 2 spaces indentation looks good.

**What actually happened? Please include the actual, raw output from ESLint.**
```bash
/home/harald/github/alexa-packer/src/test.js
  1:17  warning  'b' is defined but never used                 no-unused-vars
  5:1   warning  Expected indentation of 0 spaces but found 2  indent
```

**Comment by not-an-aardvark** [comment]
Closing as a duplicate of https://github.com/eslint/eslint/issues/9706
